lib/galaxy/managers/workflows.py

```diff
@@ -388,12 +388,12 @@ def _workflow_to_dict_run( self, trans, stored ):
                     'name'   : step.module.name,
                     'inputs' : [ input.to_dict( trans ) for input in inputs.itervalues() ]
                 }
-            step_model[ 'step_id' ] = step.id
+            step_model[ 'step_id' ] = trans.app.security.encode_id( step.id )
```

Comment:
Do you have a reason not to just use the ``order_index`` for these? The other export endpoints do this and I think we should be consistent - I also think this makes the API more "usable". I can't think of a reason the GUI should need to know the step ID when the invocation API doesn't use it and the workflow editor doesn't use it - is there one?